puma/histogram.py

```diff
@@ -21,7 +25,7 @@ def __init__(  # pylint: disable=too-many-arguments
         values: np.ndarray,
         weights: np.ndarray = None,
         ratio_group: str = None,
-        flavour: str = None,
+        flavour: str | Flavour = None,
```

Comment:
Just for my understanding: Is this somehow the new recommended way to write this?

Reply:
Indeed https://peps.python.org/pep-0604/